__mocks__/request-promise.js

```diff
@@ -0,0 +1,47 @@
+import {getConfig} from '../src/__mocks__/helpers';
+
+
+function request(url) {
+  // TODO get rid of those globals
+  switch (global.MOCK_REQUEST_SCENARIO) {
+    case 'broker-latest-pact':
```

Comment:
This is more like const value. You could put that to a separate file because you are using this in several places.